andela/wg-silicon
# [Feature #158626115] Add function to enable display of tab on hover

#### What does this PR do?
Enable getting exercises for a body part when the mouse hovers over the body part

#### Description of Task to be completed?
The user should be able to move my mouse over a body part and get exercises for that body part

#### How should this be manually tested?
- Start the server
- Navigate to exercises tab
- Hover on the body parts to see the exercises

### What are the relevant pivotal tracker stories?

[#158626115](https://www.pivotaltracker.com/story/show/158626115)

#### Screenrecording
![exercise-onhover](https://user-images.githubusercontent.com/20478530/43307657-847d6ac2-9187-11e8-9d3e-1c6a0e161b14.gif)
